src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -31,9 +32,18 @@ internal sealed class ProcessMetrics
     private double? virtualMemoryUsage;
     private double? userProcessorTime;
     private double? privilegedProcessorTime;
+    private int? numberOfThreads;
+    private IEnumerable<Measurement<double>> cpuUtilization;
+
+    // vars for calculating CPU utilization
+    private DateTime collectionTimeStamp;
+    private double collectionUserProcessorTime;
+    private double collectionPrivilegedProcessorTime;
```

Comment:
nit: I would add units to these. Makes it more maintainable in the long run IMO.

```csharp
    private DateTime collectionTimeStampUtc;
    private double collectionUserProcessorTimeSeconds;
    private double collectionPrivilegedProcessorTimeSeconds;
```

Could also do the ones above:

```csharp
    private double? memoryUsageBytes;
    private double? virtualMemoryUsageBytes;
    private double? userProcessorTimeSeconds;
    private double? privilegedProcessorTimeSeconds;
```

Reply:
adding units to the above mentioning fields as suggested; thanks!